web/asgi.py

```
@@ -14,30 +14,27 @@
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "web.settings")
-django.setup()
 
-try:  # Import websocket URL patterns (must exist when Channels is assumed installed)
-    from web.routing import websocket_urlpatterns  # type: ignore
-except Exception:  # pragma: no cover - fail safe with empty list
-    websocket_urlpatterns = []  # type: ignore
+# Initialize Django before importing anything that requires ORM
+django.setup()
 
-django_asgi_app = get_asgi_application()
+# noqa annotations silence E402 (module level import not at top of file)
```

Ruff (0.14.0)

Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)